server/server.js

```diff
@@ -10,7 +10,7 @@ app.use(express.json());
 
 app.use(function(req, res, next) {
 	res.header("Access-Control-Allow-Origin", "https://heagle.herokuapp.com"); // update to match the domain you will make the request from
-	//res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
+	res.header("Access-Control-Allow-Origin", "http://localhost:3000"); // update to match the domain you will make the request from
```

Comment:
Should both of these be uncommented at the same time?

Reply:
Yeah from my testing both can be uncommented at the same time, basically I think it is like a whitelist for certain urls

Reply:
Ok sounds good, thanks